src/templates/article.js

```diff
@@ -130,16 +135,21 @@ const Article = props => {
     );
     const meta = { authors, slug: slugWithAllSlashes, title };
     const { siteUrl } = useSiteMetadata();
-    const articleBreadcrumbs = [
-        { label: 'Home', target: '/' },
-        { label: 'Learn', target: '/learn' },
-    ];
+    let breadcrumb = [...ARTICLE_BREADCRUMB];
     if (type && type.length) {
-        articleBreadcrumbs.push({
-            label: type[0].toUpperCase() + type.substring(1),
-            target: `/type/${getTagPageUriComponent(type)}`,
-        });
+        breadcrumb = [
```

Comment:
Perhaps we could use the `splice` command https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice

This would let us initialize breadcrumb and then insert type at index 2 while deleting 0, what do you think?

Reply:
Also, perhaps we could move the const into the component to avoid the redeclaration

Reply:
Great idea, Also I will add `memo` for this logic.